src/base/server_type.cpp

```diff
@@ -252,6 +262,20 @@ bool is_tran_server_with_remote_storage ()
   return false;
 }
 
+active_tran_server *get_active_tran_server_ptr ()
+{
+  if (is_active_transaction_server ())
+    {
+      assert (ats_Gl != nullptr);
+      return ats_Gl;
+    }
+  else
+    {
+      assert (is_passive_transaction_server ());
```

Comment:
Here, same remark as the below comment.
But I might be wrong, as this is a newly added function.
```suggestion
      assert (is_active_transaction_server ());
```